Optimization.md

```diff
@@ -322,6 +322,11 @@ or getting stuck in local solutions.
     optimization on a linear objective function. It contains a number of
     algorithms to choose from and offers a formula syntax similar to
     `lm()`.
+-   `r pkg("nlsic")` solves non linear least squares problem (possibly
+    rank deficient) with linear equality/inequality constraints.
+    Linearized problems are solved as LSI (linear least squares with
+    inequality constraints), i.e. QR-based, cf. `r pkg("nnls")`.
+    Non linear iterations are globalized by backtracking method.
```

Comment:
Can you simplify this? Or we just leave it out?

Reply:
What exactly I have to simplify? The whole description, or just the mention of nnls?
Thanks for your advices.

Reply:
Sorry, just the 
`Non linear iterations are globalized by backtracking method.`

Reply:
Ah. The sentence is looking simple enough for me. And the information is important to guide someone's choice, is not it? But if, as TaskView editor, you consider that it is superfluous, you can leave it out.